paramak/parametric_reactors/ball_reactor.py

```diff
@@ -154,6 +152,9 @@ def __init__(
             self.plasma_gap_vertical_thickness,
             self.major_radius - self.minor_radius]
 
+        # calcualted internally by the class
+        self._plasma = None
```

Comment:
This was a suggestion of code-inspector but it appears to have broken the tests. Any ideas appreachiated

Reply:
```suggestion
        self.plasma = None
```

Reply:
Having _ before attributes hides them for outside the instance. Except if these are defined in __init__. Therefore, initialising self._plasma here will enable the access of the attribute from outside the instance and make the test fail (the test checks that all the attributes with _ are hidden)

Reply:
Good to know, wondering if we should rename _plasma to plasma then?

Reply:
Up to you. It depends on the reason it was hidden in the first place. If hidden variables are required, I'd suggest to remove all underscores for simplicity